docs/getting_started/installation.md

```diff
@@ -79,10 +79,13 @@ N rows in set (0.00 sec)
 ~~~
 
 !!! Note
-    MySQL v5.7 uses a default transaction isolation level of `REPEATABLE-READ`. You'll need to change this to `READ-COMMITTED` so that iRODS operates properly. This setting needs to be set globally. One way to achieve this is by starting MySQL with `--transaction-isolation=READ-COMMITTED`. For more information, see [https://dev.mysql.com/doc/refman/5.7/en/innodb-transaction-isolation-levels.html](https://dev.mysql.com/doc/refman/5.7/en/innodb-transaction-isolation-levels.html).
-    
+    iRODS is incompatible with versions 8.1, 8.2, and 8.3 of the MySQL ODBC Connector. You'll need to use v8.0 or v8.4.
```

Comment:
versions -> "unreleased versions" ?

Reply:
Where does it say they're unreleased?
Is it because they aren't considered GA (General Availability)?

Reply:
Yeah they're released. They're just not considered stable or GA.

Reply:
Ah, then carry on.  Resolving.